projects/ngx-formentry/src/components/machine-learning-button/model-helper.ts

```diff
@@ -328,18 +330,22 @@ export function generatePredictionPayload(
 
   // convert months since last test
 
-  if (monthsSinceLastTestInt > 0) {
-    if (monthsSinceLastTestInt <= 6) {
-      predictionVariables.MonthsSinceLastTestLASTSIXMONTHS = 1;
-    } else if (monthsSinceLastTestInt >= 24) {
-      predictionVariables.MonthsSinceLastTestMORETHANTWOYEARS = 1;
-    } else if (monthsSinceLastTestInt >= 7 && monthsSinceLastTestInt <= 12) {
-      predictionVariables.MonthsSinceLastTestSEVENTOTWELVE = 1;
-    } else if (monthsSinceLastTestInt >= 12 && monthsSinceLastTestInt <= 24) {
-      predictionVariables.MonthsSinceLastTestONETOTWOYEARS = 1;
+  if(testHistory == 1065) { // Tested for HIV before?
```

Comment:
It would be nice to introduce variables for the concepts. Right now one has to know what they mean.

Reply:
Ok. On it.